package Exception_Handling;

public class Sample8 {
public static void main(String[] args) {
	
	try 
	{
		System.out.println("running try block");
		System.out.println(10/1);
	} 
	catch (Exception e) {
		// TODO: handle exception
		System.out.println("exception handled");
	}
	
	
}
}
